src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -391,19 +420,20 @@ impl Default for MDAHeader {
 
 impl MDAHeader {
     /// Parse a valid MDAHeader from buf.
-    /// If the timestamp value is 0, then return None. This means that
-    /// No variable length metadata has been written, so there is no
-    /// corresponding header.
+    /// If the amount used by the variable length metadata is 0, return None,
+    /// as this means that no variable length metadata has been written.
     fn parse_buf(buf: &[u8; mda_size::_MDA_REGION_HDR_SIZE]) -> Option<MDAHeader> {
-        match LittleEndian::read_u64(&buf[16..24]) {
+        match LittleEndian::read_u64(&buf[8..16]) {
```

Comment:
Is it normal that it changed from `16..24` to `8..16`?

Reply:
Exactly!